CMakeLists.txt

```diff
@@ -276,19 +306,43 @@ if(OPENVDB_BUILD_DOCS)
   endif()
 
   # @todo use cmake doxygen functions available from cmake 3.9
-  file(READ ${CMAKE_CURRENT_SOURCE_DIR}/openvdb/doxygen-config DOXYGEN_CONFIG_CONTENT)
+  file(READ ${CMAKE_CURRENT_SOURCE_DIR}/doc/doxygen-config DOXYGEN_CONFIG_CONTENT)
   file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/openvdb/cmake-doxygen-config ${DOXYGEN_CONFIG_CONTENT})
+  if(OPENVDB_BUILD_HOUDINI_PLUGIN)
+    # Append Houdini-specific settings to the Doxygen config file.
+    file(READ ${CMAKE_CURRENT_SOURCE_DIR}/doc/doxygen-config-houdini DOXYGEN_CONFIG_CONTENT)
+    file(APPEND ${CMAKE_CURRENT_BINARY_DIR}/openvdb/cmake-doxygen-config ${DOXYGEN_CONFIG_CONTENT})
+    if(Houdini_VERSION)
+      # Extract the components of the Houdini version number into a list.
+      string(REPLACE "." ";" HOUDINI_VERSION_COMPONENTS ${Houdini_VERSION})
+      list(GET HOUDINI_VERSION_COMPONENTS 0 HVERS_MAJOR)
+      list(GET HOUDINI_VERSION_COMPONENTS 1 HVERS_MINOR)
```

Comment:
This change required me to move the DCC configuration section up to ensure that `Houdini_VERSION` is defined.  I'm not sure whether it's safe to do that—@Idclip, can you comment?

Reply:
Moving the DCC configuration to here is fine, the docs previously didn't depend on the houdini or maya install so if you just wanted to build a cmake configuration with the docs in then you could get unrelated errors if it couldn't find those installations. 

They still don't rely on the maya installation, so a configuration with all build options off which uses `OPENVDB_BUILD_DOCS=ON USE_HOUDINI=ON USE_MAYA=ON` will fail due to the `if(USE_MAYA AND USE_HOUDINI)` build catch. Perhaps move that branch to after the early build exit? (line 354) This will support any future behaviour where any maya docs which rely on the maya version.